### DISCUSSION TITLE
New pattern for Pannier

### DISCUSSION THREAD

**Discussion by darigovresearch** [freesewing/freesewing#3461] [Forum]
Raised on the contributor call on 21 January 2023. Can be made from upcycled material like torn sail, tarp or tent canvas and can be used as further material to make other freesewing garments if you make a mistake or the pannier becomes worn out.

https://en.wikipedia.org/wiki/Pannier

Would suggest Pain the pannier (Pain means bread in French)

**Comment by joostdecock** [comment]
Closing to give room to active discussions

**Comment by darigovresearch** [comment]
Reopening discussion from the community call https://github.com/freesewing/freesewing/discussions/5532

**Comment by BenJamesBen** [comment]
Closing because there does not seem to be any interest for active discussion about this topic, per the April 27, 2024 Community Call. https://github.com/freesewing/freesewing/discussions/5724#discussioncomment-9247197

(Please feel welcome to re-open this Discussion if there is interest in resuming active discussion about the topic.)
